examples/request-certificate/main.go

```diff
@@ -85,10 +85,13 @@ func NewClient(fulcioURL string) (fulciopb.CAClient, error) {
 		return nil, err
 	}
 	dialOpt := grpc.WithTransportCredentials(insecure.NewCredentials())
+	hostWithPort := fmt.Sprintf("%s:80", fulcioServer.Host)
 	if fulcioServer.Scheme == "https" {
 		dialOpt = grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{}))
+		hostWithPort = fmt.Sprintf("%s:443", fulcioServer.Host)
 	}
-	conn, err := grpc.Dial(fulcioServer.Host, dialOpt)
+
+	conn, err := grpc.Dial(hostWithPort, dialOpt)
 	if err != nil {
 		return nil, err
```

Comment:
While you're updating this, could you also change the default signer to ECDSA? If people copy this, I'd prefer that over RSA.